config/lenovo-a6020a40.dumb.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From AospExtended-v7.2-A6020-20200513-2005-OFFICIAL.zip
+
+OPTIONS=(-r vsp -r vsn --dumb-dcs)
```

Comment:
What is the rule behind this `--dumb-dcs` flag, should it be used whenever suggested by the script or only when necessary?

Reply:
If there is more than a couple WARNING lines that suggest using `--dumb-dcs` it should typically be used. AFAICT in this case it is

```
WARNING: DCS command SOFT_RESET with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command ENTER_SLEEP_MODE with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command EXIT_SLEEP_MODE with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command ENTER_PARTIAL_MODE with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command ENTER_NORMAL_MODE with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command EXIT_INVERT_MODE with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command ENTER_INVERT_MODE with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command SET_DISPLAY_OFF with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command SET_DISPLAY_ON with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command SET_COLUMN_ADDRESS with incorrect argument count (expected: 4, is: 1). Consider using --dumb-dcs
WARNING: DCS command SET_PAGE_ADDRESS with incorrect argument count (expected: 4, is: 1). Consider using --dumb-dcs
WARNING: DCS command SET_SCROLL_AREA with incorrect argument count (expected: 6, is: 1). Consider using --dumb-dcs
WARNING: DCS command SET_TEAR_OFF with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command SET_SCROLL_START with incorrect argument count (expected: 2, is: 1). Consider using --dumb-dcs
WARNING: DCS command EXIT_IDLE_MODE with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command ENTER_IDLE_MODE with incorrect argument count (expected: 0, is: 1). Consider using --dumb-dcs
WARNING: DCS command SET_TEAR_SCANLINE with incorrect argument count (expected: 2, is: 1). Consider using --dumb-dcs
```

so I agree that it makes sense here. Otherwise the generated panel driver would look strange.

You can usually see it if the panel driver has a large chunk of `mipi_dsi_dcs_write_seq()` with varying numbers. This suggests that they are (ab)using standard DSI DCS commands to write to vendor-specific registers. The `--dumb-dcs` option makes the generator aware of that and prevents it from incorrectly interpreting those commands.